app/controllers/admin/api/service_contracts_controller.rb

```diff
@@ -8,6 +8,7 @@ class Admin::Api::ServiceContractsController < Admin::Api::ServiceBaseController
   before_action :authorize_service_plans!
 
 
+  ##~ sapi = source2swagger.namespace("Account Management API")
```

Comment:
I know that this seems unrelated to this PR but the reason is because if it is undefined, it goes to the `Policy Registry API` when we generate the docs 😂

Reply:
In https://github.com/3scale/porta/pull/605 now 😄